django/src/rdwatch_scoring/router.py

```diff
@@ -31,7 +31,7 @@ def db_for_write(self, model: type[Model], **hints) -> DbName | None:
         """
         if model in RGD_DB_MODELS or model._meta.app_label == 'rdwatch':
             return 'default'
-        return None
+        return 'scoringdb'
```

Comment:
Could we reverse this to be excplicit when `model._meta.app_label === 'rdwatch_scoring'` like the db_for_read about  I would want the default to be the 'default'